tests/unittests/core/test_trial.py

```diff
@@ -156,65 +156,143 @@ def test_invalid_result(self, exp_config):
                          Trial.Result(name='asfdb', type='objective', value=None)]
         assert 'Results must contain' in str(exc.value)
 
-    def test_objective_property(self, exp_config):
+    def test_objective_property(self):
         """Check property `Trial.objective`."""
+        base_trial = {
+            'results': []
+        }
+        base_trial['results'].append({'name': 'a', 'type': 'gradient', 'value': 0.5})
+
+        # 0 objective in `results` list
+        trial = Trial(**base_trial)
```

Comment:
May be worth simplifying? Dunno... :man_shrugging: 
```suggestion
        trial = Trial(results=[{'name': 'a', 'type': 'gradient', 'value': 0.5}])
```

Reply:
Ah, append later on. Forget it.